pkg/process/filter_clusters.go

```diff
@@ -0,0 +1,40 @@
+package process
+
+import (
+	"github.com/kyma-project/kyma-environment-broker/common/runtime"
+	"gopkg.in/yaml.v3"
+	"os"
+)
+
+type accounts struct {
+	MeteringAccounts []string `yaml:"meteringAccounts"`
```

Comment:
lets have this more descriptive:
eg: Skipped<Global|Sub>AccountIDs (also in the yaml)